ui/lib/kv/addon/components/kv-patch/editor/form.hbs

```diff
@@ -15,7 +15,7 @@
 
   {{! Rows for existing keys (includes new rows after user clicks "Add") }}
   {{#each this.patchData as |kv idx|}}
-    <KvPatchEditor::Row
+    <KvPatch::Editor::Row
```

Comment:
I moved all of the `kv-patch` components into a directory and everything related to the "editor" which is the UI key/value form into a folder. Hopefully this clarifies which component is for what, but open to suggestions on improving the organization!